_health-care/_js/components/form-elements/ErrorableTextInput.jsx

```diff
@@ -43,7 +43,7 @@ class ErrorableTextInput extends React.Component {
     }
 
     return (
-      <div className={`usa-input-grid usa-input-grid-medium ${this.props.errorMessage ? 'usa-input-error' : ''}`}>
+      <div className={`${this.props.errorMessage ? 'usa-input-error' : ''}`}>
```

Comment:
In this case, you don't need a template string anymore, and you can change '' to undefined so no class attribute is even generated.